### DISCUSSION TITLE
Add mod advertising.

### DISCUSSION THREAD

**Discussion by ghost** [DEMOLITIONDON96/Demolition-Engine#111] [Polls]
As long as this isn't spammed like crazy, Demo may be up to doing it.

Might even let some mods be pinned for a while :trollface:

**Comment by ghost** [comment]
Ahem.... Lets ALSO talk about v0.3!

**Reply by RealDealJinx** [reply]
i was gonna add the option to have the shitty health drain and stuff but fuck
i didnt save the playstate file and closed it so progress needs to be done again

**Reply by RealDealJinx** [reply]
also v0.3 may have a redesgin menu
fuck load of new options
and may fix the freeplay thing demo wanted

**Reply by ghost** [reply]
EEE

**Comment by ThatOneSillyGuyDon** [comment]
no. Only because the max categories you can put is 10, and GitHub won't let me add more than that

**Reply by ghost** [reply]
Oh.
Well what the heck do we need Ideas (Softcode) when we can always combine them into one. :trollface: 

I'm kidding. I figure everybody would spam the heck out of them anyways regardless of our warnings.
